### ISSUE TITLE
parse_files.py:72: ComplexWarning: Casting complex values to real discards the imaginary part

### ISSUE THREAD

**Issue by piteight** [MattVitelli/GRUV#25]
Hello, I have trained model and generated sample mp3, but when I was using generate.py this happened:

/GRUV/data_utils/parse_files.py:72: ComplexWarning: Casting complex values to real discards the imaginary part
  Xnew = Xnew.astype('int16')

I dont know what it mean, the output song is something like random noise + beat, but it is 50 iter, so it has to be noisy.
Nobody had that issue, its confusing.

**Comment by YouthBread** [comment]
I got this error too and i dont know how to fix it.
And I've modified some of the code in order to use updated keras, but still stucked on generating.
`seedSeq = np.concatenate((seedSeq,newSeq),axis=1)`
cmd returns 

> Exception: Error when checking: expected timedistributed_input_1 to have shape (None,20,4096) but got array with shape(1,21,4096)

you got any idea?

**Comment by piteight** [comment]
mattpearson made version compatible with newest keras, please look at  https://github.com/mattpearson/GRUV
I have noticed that in that version, it switched to tensorflow beckend and used all CPU cores, not 1 like it was in older version. Installing tensorflow with cuda 8.0 support get GPU working.
